frontend/packages/kubevirt-plugin/src/components/vms/vmi-details-page.tsx

```diff
@@ -22,11 +22,10 @@ import { VMConsoleFirehose } from './vm-console';
 import { VMDetailsFirehose } from './vm-details';
 import { vmiMenuActionsCreator } from './menu-actions';
 import { VMDashboard } from './vm-dashboard';
+import InfoMessageHintBlock from '../info-messages/vmi-details-page-info-message';
```

Comment:
wooo ...

Reply:
missed the typo here, cancel the lgtm

p.s. we also need to change the file names to camel-case (e.g. `VMIDetailsPageInfoMessage.tsx` and  `VMIDetailsPageInfoMessage.spec.tsx`)